### PR TITLE
OPS: updated payjoin dep

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pg8j21t87jw38hu44d0vb599fc

[download apk](http://filestorage.bluewallet.io:1488/1606235724.apk)

File: screen/send/confirm.js [hunk 1]
```diff
@@ -46,6 +46,18 @@ export default class Confirm extends Component {
...
+   * we need to look into `recipients`, find destination address and return its outputScript
+   * (needed for payjoin)
+   *
+   * @return {string}
+   */
+  getPaymentScript() {
+    for (const recipient of this.state.recipients) {
+      return bitcoin.address.toOutputScript(recipient.address);
+    }
+  }
```
Comment (limpbrains) [inline screen/send/confirm.js:59]: You just take the first value from the recipients list and evaluate the outputscript.  are you sure that this is the destination address?
Looks like this function doesn't match it's description.

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: screen/send/confirm.js (hunk 1 above)
Comment (Overtorment) [reply screen/send/confirm.js:59]: pretty sure. in payjoin, there is only one destination. in batch send payjoin wont work.
plus, I tested it on real payjoin-enabled server  https://btc.donate.kukks.org

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**